radio@hslbck.gmail.com/addChannelDialog.js

```diff
@@ -186,13 +182,12 @@ function getStreamAddress(input) {
             for (var line in contentLines) {
                 if (contentLines[line].search(/http:/i) != -1) {
                     // get url
-                    streamAddress = contentLines[line].slice((contentLines[line].search(/http:/))) //, contentLines[line].search(/\n/));
-                    break;  // break on the first occurrence
+                    return contentLines[line].slice((contentLines[line].search(/http:/))) //, contentLines[line].search(/\n/));
                 }
             }
         }
-    } else {
-        streamAddress = input; // case for valid stream address
+        return; // TODO: ?
```

Comment:
@hslbck This case was not explicit before. Now, I've got to write it, otherwise we would return `input`. As it was not handled before, I think we'll get the same result with this `return;` but I'm not sure as I don't know JavaScript ; but maybe we should raise an exception or print an error ?